src/t8_forest/t8_forest.cxx

```diff
@@ -1414,119 +1417,82 @@ t8_forest_copy_trees (t8_forest_t forest, t8_forest_t from, int copy_elements)
   }
 }
 
```

Comment:
I simplified the t8_forest_element_neighbor_eclass  function.
It works with elements and ghosts now.